seqr/management/commands/reload_clinvar_all_variants.py

```diff
@@ -66,7 +66,6 @@ def clinvar_run_sql(sql: str):
         ]:
             cursor.execute(sql.substitute(reference_genome=reference_genome, dataset_type=dataset_type))
```

Comment:
not particularly related to this PR, but this file should really be in the `clickhouse_search/management/commands` folder not the seqr commands folder